### PR TITLE
Add pylint checks to travis

### PR DESCRIPTION
**Pull request by mwidera**
Also fix couple of errors for both python versions

Change-Id: Id274d4b4378e015fd2d8796a753c31e8204d4b3f
Signed-off-by: Michal Widera <michal.widera@arm.com>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -24,9 +24,10 @@ before_install:
   - sudo apt-get update
   - |
     sudo apt-get install -y rsync python3 \
-     python-ply  python-pytest  python-pytest-catchlog  python-mock  python-pytest-mock \
+    python-ply  python-pytest  python-pytest-catchlog  python-mock  python-pytest-mock \
```
Comment (kilroyd) [inline .travis.yml]: I think Chris was aligning the lines on the '-' so we can see we have the same things installed for python2 and python3. Can you keep to that please?

Comment (mwidera) [reply .travis.yml]: Ok changed, so requirements are now in aligned

**Comment by kilroyd** [comment]
The test failure looks to be due to a change in the way travis checks out the PR commit. Please ignore the bootstrap test failure which we'll fix separately

File: config_system/config_system/general.py [hunk 2]
```diff
@@ -756,11 +756,7 @@ def get_default_style():
 
 
 class StyledText(object):
-    def __init__(self):
-        self.style = get_default_style()
-        self.text = ""
-
-    def __init__(self, text):
+    def __init__(self, text=""):
```
[the remark this answers is not in the document]
Comment (mwidera) [reply config_system/config_system/general.py]: It is just a merge of duplicated constructors for this class. After consultation default value is not needed anymore, thus it was removed (see latest push)

**Pull request merged**